phys2bids/interfaces/txt.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+phys2bids interface for txt files.
+"""
+from bioread import read_file
+
+from phys2bids.physio_obj import BlueprintInput
+
+import numpy as np
```

Comment:
Can you invert import of numpy and import of phys2bids?
It's more similar to the rest of the code (external mod first, then internal mod)